runtime/src/backends/dynamodb.rs

```diff
@@ -0,0 +1,448 @@
+use anyhow::Result;
+use async_trait::async_trait;
+use bytes::Bytes;
+use core::borrow::BorrowMut;
+use hyper::{HeaderMap, Response, StatusCode};
+use rusoto_core::{Region, RusotoError};
+use rusoto_dynamodb::{
+    AttributeValue, DeleteItemError, DeleteItemInput, DeleteItemOutput, DynamoDb, DynamoDbClient,
+    GetItemError, GetItemInput, GetItemOutput, PutItemError, PutItemInput, PutItemOutput,
+};
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
+use std::{collections::HashMap, env};
+
+extern crate flexbuffers;
+
+use crate::storage::{Item, ItemKey, Storage};
+
+pub struct DynamoDbCacheClient {
```

Comment:
Could we rename the struct to `DynamodDbBackend` to follow the naming convention for backends.